mmc-github/src/main/java/info/novatec/metricscollector/github/GithubRepository.java

```diff
@@ -38,22 +35,22 @@ void saveMetrics(GithubMetrics metrics) {
         influx.savePoint(createPoints(metrics));
         influx.close();
         log.info("Added point  for '" + metrics.getRepositoryName() + "' to InfluxDb Measurement '"
-            + metrics.getRepositoryName() + "'.");
+                + metrics.getRepositoryName() + "'.");
```

Comment:
Why are there 8 trailing spaces? There should be just 4 of them. Its maybe due to the fact, that your idea settings are different to mine. I will checkin a formatter for eclipse and intellij idea.